### ISSUE TITLE
About  Real-World Degradation？

### ISSUE THREAD

**Issue by dreamhua82** [YingqianWang/LF-InterNet#12]
Hi, how do you get the  Real-World Degradation samples? It is not using bicubic downsampling method?
(2) How do you retrained the ResLF? 
Thans a lot.

**Comment by YingqianWang** [comment]
For real-world SR, we directly applied SR methods to original HR images without any downsampling operation. Since the groundtruth HHR images are unavailable, we only compared the visual results of different methods in our paper. BTW, we directly used the model trained under bicubic degradation for real SR without any finetuning or modification.

For the retraining of resLF, we have followed the implementation details in their paper and tried our best to maximize their reconstruction accuracy.
